Split/Storage/SplitDatabase.swift

```diff
@@ -63,57 +62,14 @@ class CoreDataSplitDatabase: SplitDatabase {
     private let kDataModelName = "split_cache"
     private let kDataModelExtentsion = "momd"
     private let kDatabaseExtension = "sqlite"
```

Comment:
Since these constants were moved to `CoreDataHelperBuilder`, check if they can be removed here.

Reply:
Yeah, done doc!